### PR TITLE
NeMo-UX: Mistral/mixtral peft ci test

### PR DESCRIPTION
**Pull request by akoumpa**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: .github/workflows/cicd-main.yml [hunk 1]
```diff
@@ -4320,6 +4320,53 @@ jobs:
         --pp_size 1 \
         --mbs 1 --packed
 
+
+  L2_NeMo_2_Mixtral_LoRA_TP1PP1_MBS1:
```
[the remark this answers is not in the document]
Comment (akoumpa) [reply .github/workflows/cicd-main.yml:4324]: Thanks for catching this.

File: .github/workflows/cicd-main.yml [hunk 2]
```diff
@@ -4320,6 +4320,53 @@ jobs:
...
+        --devices 2 \
+        --max_steps 3 \
+        --tp 1 \
+        --mbs 1 \
+        --model mistral
+
+  L2_NeMo_2_Mistral_LoRA_TP2PP1_MBS1:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_Mistral_LoRA_TP1PP1_MBS1') || needs.cicd-test-container-setup.outputs.all == 'true'
```
Comment (ko3n1g) [inline .github/workflows/cicd-main.yml:4357]:
```suggestion
    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_Mistral_LoRA_TP2PP1_MBS1') || needs.cicd-test-container-setup.outputs.all == 'true'
```

Comment (ko3n1g) [reply .github/workflows/cicd-main.yml:4357]: I should add an automatic check for verifying these typos

**Review by ko3n1g — CHANGES_REQUESTED** [review]
one more and we're good to go

**Review by ko3n1g — APPROVED** [review]

**Review by ko3n1g — APPROVED** [review]

**Pull request merged**